README.md

```diff
@@ -49,7 +49,7 @@ ORC heavily uses [CEL validations](https://kubernetes.io/docs/tasks/extend-kuber
 To install a released version of ORC, the simplest is probably to use the provided kustomization file:
 
 ```
-export ORC_RELEASE="https://github.com/k-orc/openstack-resource-controller/dist?ref=v2.0.2"
```

Comment:
Won't that apply from main? I suspect that won't work in all cases, for example if we've had to create a release branch for the latest release.

Reply:
> Won't that apply from main? 

That's indeed the idea, but I haven't tried to check if it works as I expect.

> I suspect that won't work in all cases, for example if we've had to create a release branch for the latest release.

Right, I kinda assume we wouldn't do that, and instead always tag the latest release from main, where the kustomize file would then always point to the latest and greatest.